msal-dotnet-articles/advanced/powershell-support.md

```diff
@@ -9,7 +9,7 @@ There is no official PowerShell module or wrapper for MSAL libraries maintained
 
 Consider using higher level APIs which are officially supported: 
 
- -  [Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started)
- -  [Azure PowerShell SDK](/powershell/azure/new-azureps-module-az)
+ -  [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell?tab=readme-ov-file)
```

Comment:
```suggestion
 -  [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell)
```

this seems to be landing on the read me tab.

Reply:
fixed